MobileWeather/app/src/main/AndroidManifest.xml

```diff
@@ -56,23 +51,37 @@
         </service>
         <service android:name="com.ford.mobileweather.wunderground.WUndergroundService" >
         </service>
-        <service android:name="com.ford.mobileweather.noaa.NOAAService" >
-        </service>
+        <!--<service android:name="com.ford.mobileweather.noaa.NOAAService" >-->
+        <!--</service>-->
```

Comment:
Is there a reason this is commented out?

Reply:
Not really :) I will just remove it completely.